net-libs/qxmpp/qxmpp-1.0.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="A cross-platform C++ XMPP client library based on the Qt framework"
+HOMEPAGE="https://github.com/qxmpp-project/qxmpp/"
+SRC_URI="https://github.com/${PN}-project/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc opus +speex test theora vpx"
```

Comment:
You can add a `static-libs` option with https://github.com/qxmpp-project/qxmpp/blob/master/src/CMakeLists.txt#L170

Reply:
personally, I don't add static-libs unless there is a request for it.

Reply:
Yep, I guess I'll go without those, especially given that `static-libs` aren't that common in Qtland (AFAICT).